src/tools/coding_speed.js

```diff
@@ -29,6 +29,7 @@ argv.erase = Boolean(argv.decode && (argv.erase !== false)); // default is true
 argv.ec = Boolean(argv.ec); // default is false, use --ec
 argv.md5 = Boolean(argv.md5); // default is false, use --md5
 argv.sha256 = Boolean(argv.sha256); // default is false
+argv.md5_mb = Boolean(argv.md5_mb); // default is false, use --md5_mb
```

Comment:
What happens if we are passing argv.md5 and argv.md5_mb? 
We can change that from 2 flags to one enum

Reply:
This was changed so that the mode is detected in src/util/fips. Resolving.